draft-ietf-quic-tls.md

```diff
@@ -103,90 +103,118 @@ benefits QUIC provides would be realized due to the handshake latency in
 versions of TLS prior to 1.3.
 
 
-## Notational Conventions
+# Notational Conventions
 
 The words "MUST", "MUST NOT", "SHOULD", and "MAY" are used in this document.
 It's not shouting; when they are capitalized, they have the special meaning
 defined in {{!RFC2119}}.
 
+This document uses the terminology established in {{QUIC-TRANSPORT}}.
+
+For brevity, the acronym TLS is used to refer to TLS 1.3.
+
+This document uses TLS terminology is used when referring to parts of TLS.
```

Comment:
I think two possible phrasings are combined here.  "This document uses TLS terminology" / "TLS terminology is used" -- but not both.